dev/script/setup.sh

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR=${DIR}/../..
+
+# load values from .env into environment variables
+# see https://stackoverflow.com/a/30969768/3868241
+set -o allexport
+# shellcheck source=../.env
+source "$ROOT_DIR/dev/.env"
+set +o allexport
+
+# ---- START static variables that shouldn't be changed
+export AWS_PAGER=""
```

Comment:
V2 of the AWS CLI uses `less` for pagination. Set it to `""` to make scripting easier.

See https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-pagination.html#cli-usage-pagination-clientside